Jetcaster/wear/src/main/java/com/example/jetcaster/ui/home/HomeScreen.kt

```diff
@@ -130,8 +138,8 @@ fun HomeScreen(
 
         content = {
             if (viewState.podcastCategoryFilterResult.topPodcasts.isNotEmpty()) {
-                val podcast = viewState.podcastCategoryFilterResult.topPodcasts.first()
-                items(viewState.podcastCategoryFilterResult.topPodcasts.take(1).size) {
+                items(viewState.podcastCategoryFilterResult.topPodcasts.take(3).size) { index ->
```

Comment:
if you remove `.size` you can pass the podcast in directly to the lambda, instead of index.

Reply:
thanks